host/3.0/core-tools-publish.yml

```diff
@@ -63,7 +63,7 @@ stages:
           IMAGE_NAME=azurefunctions.azurecr.io/azure-functions/3.0/dotnet:$(PrivateVersion)-dotnet3-core-tools
     
           docker build -t $IMAGE_NAME \
-                      -f host/3.0/buster/amd64/dotnet/dotnet-core-tools.Dockerfile \
+                      -f host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-core-tools.Dockerfile \
                       host/3.0/buster/amd64/dotnet/
```

Comment:
would this change to `host/3.0/buster/amd64/dotnet/dotnet-inproc/`?

Reply:
Done. Very good catch